carbon-design-system/carbon-for-ibm-dotcom
# fix(featured-section): adjust target new tab issue by extending dds-card-cta

### Related Ticket(s)

DDS Consulting Issue: https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues/7072
HC JIRA Ticket: https://jsw.ibm.com/browse/HC-2217

### Description
- This PR extends `dds-card-cta` for the `dds-feature-section-card-link` used in the CTA card area, so that adopters can define the CTA type, etc.

### Testing Instructions
![Screen Shot 2021-09-24 at 3 14 48 PM](https://user-images.githubusercontent.com/1815714/134728504-a2e03630-b20b-40f0-9138-62e94f16f5b3.png)



- Visit the `Feature Section` layout Component - `?path=/story/components-feature-section--default`
- Adjust the `CTA Type` to `external`.
- Confirm that the `target="_blank"` attribute is added to the shadow DOM of the `<a>` link and that the CTA opens into a new tab.
